test-packages/framework/api.d.ts

```diff
@@ -6,3 +6,5 @@ export {
   ReadFileResult,
   TestFramework,
 } from "./src/api";
+
+export const CURSOR_ANCHOR;
```

Comment:
maybe simply remove from here and added after line 7

like this consumer see it like
```typescript
(alias) const CURSOR_ANCHOR: any
```
where it should be

```typescript
(alias) const CURSOR_ANCHOR: "⇶"
```

Reply:
done